Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/CharacterEmoteSystem.cs

```diff
@@ -181,10 +181,9 @@ private void ConsumeEmoteIntent(Entity entity, ref CharacterEmoteComponent emote
                     World.Remove<CharacterEmoteIntent>(entity);
                 }
             }
-            catch (Exception e)
-            {
-                ReportHub.LogException(e, reportCategory);
-            }
+            catch (Exception e) { ReportHub.LogException(e, reportCategory); }
+
+            World.Remove<CharacterEmoteIntent>(entity);
```

Comment:
I wonder why you added it here? Now if it emote was not cached (still downloading) the intention will be lost?

Reply:
Weird that it shows here as a diff, as this is a from a merge from Pato from last month? 
maybe I fucked up the merge or git didnt see any conflict? 🤔
And I see that as of yesterday the World.Remove is inside the try...
I will fix this :) thank you for spotting it!!